### PR TITLE
Add LoadKey

### PR DESCRIPTION
**Pull request by alex-miller-0**
Adds function path to load a key or seed onto the card.

Extends #4

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -207,6 +209,60 @@ func NewCommandDeriveKey(pathStr string) (*apdu.Command, error) {
...
+		p1 = 0x02
+	} else {
+		p1 = 0x01
+	}
+	return apdu.NewCommand(
+		globalplatform.ClaGp,
+		InsLoadKey,
+		p1,
+		0,
+		data,
```
Comment (gravityblast) [inline commands.go:231]: if we are loading a keypair or extended keypair, we need to format the data with the TLV keypair template, looking at the documentation only with `P1=0x03` we can send raw data.

**Comment by gravityblast** [comment]
hey @alex-miller-0 do you want to rebase and continue with this PR? otherwise there's no problem, I can continue the work if you don't have time. and thank you for your contributions :)

File: commands.go (hunk 1 above)
Comment (alex-miller-0) [reply commands.go]: Ah, so you just want it to block sending data if `p1 != 0x03`?

**Comment by alex-miller-0** [comment]
Our fork has actually diverged from this PR so it should probably be closed.

Here's the branch where we settled: https://github.com/alex-miller-0/keycard-go/tree/add-load-key

I can open a new PR with that one instead if you would like. Should still be compatible with your codebase.

File: commands.go (hunk 1 above)
Comment (alex-miller-0) [reply commands.go:231]: Good catch. I believe f2947d9 should do what you want

**Comment by alex-miller-0** [comment]
@gravityblast apologies, I must have missed your review! Check out the updated code.

**Comment by alex-miller-0** [comment]
Having trouble rebasing - will open a new PR

**Pull request closed without merging**